cloud/cloud.go

```diff
@@ -93,21 +94,24 @@ func getVirtualMachine(username string) string {
 	return resp.VMHostname
 }
 
-func syncFiles(username string, hostname string) error {
+func syncFiles(username string, hostname string, box *devbox.Devbox) error {
+	dirName := projectDirName(box.GetConfigDir())
+	debug.Log("Will sync files to directory: %s", dirName)
+
 	// TODO: instead of id, have the server return the machine's name and use that
 	// here to. It'll make things easier to debug.
 	id, _, _ := strings.Cut(hostname, ".")
 	_, err := mutagen.Sync(&mutagen.SessionSpec{
 		// If multiple projects can sync to the same machine, we need the name to also include
 		// the project's id.
 		Name:        fmt.Sprintf("devbox-%s", id),
-		AlphaPath:   ".", // We should use location of `devbox.json`
+		AlphaPath:   box.GetConfigDir(),
 		BetaAddress: fmt.Sprintf("%s@%s", username, hostname),
 		// It's important that the beta path is a "clean" directory that will contain *only*
 		// the projects files. If we pick a pre-existing directories with other files, those
 		// files will be synced back to the local directory (due to two-way-sync) and pollute
 		// the user's local project
-		BetaPath:  "~/Code/",
+		BetaPath:  "~/Code/", // Use ~/Code/{dirName}, where dirName is the variable above
```

Comment:
We'll sync to `~/{dirName}`. No need for `Code/` prefix.

Reply:
I kinda like:
`~/code/<project>` or `~/project/<project>` in case we ever want to sync up multiple projects to the VM, and keep your home directory "uncluttered"

FWIW, this is what stackblitz does: 
`~/projects/nuxt-starter-klkvsl`

Reply:
okay okay, I don't feel strongly either ways, so lets go with `~/code/<project>`